Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -47,18 +47,6 @@ For more precisely positioning your hologram, you can use additional input from
 
 <br>
 
-
-[**Visualization of Visual Attention**](EyeTracking_Visualization.md)
-
-Information about where users look at is an immensely powerful tool to assess usability of a design and to identify problems in efficient work streams. 
-This tutorial discusses different eye tracking visualizations and how they fit different needs. 
-We provide basic examples for logging and loading eye tracking data and examples for how to visualize them. 
-
-**Summary**: Two-dimensional attention map (heatmaps) on slates. Recording & replaying eye tracking data.
```

Comment:
This sample does exist: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/prerelease/2.0.0_stabilization/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-05-Visualizer.unity

Reply:
I can keep this blurb around, but just delete the link